predicthq/endpoints/v1/events/schemas.py

```diff
@@ -59,6 +59,57 @@ class Options:
     formatted_address = StringType()
 
 
+class Point(Model):
+
+    type = StringType()
+    coordinates = ListType(FloatType())
+
+    @classmethod
+    def _claim_polymorphic(cls, data):
+        return data.get('type') in ['Point']
+
+
+class MultiPoints(Model):
```

Comment:
Any reason this one is plural? Class/Model are generally singular.

Reply:
updated